docs/reference/settings/audit-settings.asciidoc

```diff
@@ -18,10 +18,17 @@ Set to `true` to enable auditing on the node. The default value is `false`.
 `xpack.security.audit.outputs`::
 Specifies where audit logs are output. For example: `[ index, logfile ]`. The
 default value is `logfile`, which puts the auditing events in a dedicated
-`<clustername>_access.log` file on the node. You can also specify `index`, which
-puts the auditing events in an {es} index that is prefixed with
-`.security_audit_log`. The index can reside on the same cluster or a separate
-cluster.
+file named `<clustername>_audit.log` on each node.
+You can also specify `index`, which puts the auditing events in an {es} index
+that is prefixed with `.security_audit_log`. The index can reside on the same
+cluster or a separate cluster.
+
+For backwards compatibility reasons, the logfile contents are duplicated to
+the `<clustername>_access.log` file with the format for versions prior to 6.5.0.
+If the backwards compatible format is not required, it should be disabled.
```

Comment:
I found the "For backwards compatibility..." sentence a bit confusing.  Maybe it would be clearer like this?:
"For backwards compatibility reasons, if you use the logfile output type, a `<clustername>_access.log` file is also created. It contains the same information, but it uses the older (pre-6.5.0) formatting style. If the backwards-compatible format..."